doc/DEVELOPMENT.rst

```diff
@@ -229,6 +231,22 @@ To modify the 3D models, you'll need `Blender <https://blender.org>`__, which is
 free and open source. Look for tutorials on YouTube for how to use it. The
 models themselves are located in ``resources/00-taisei.pkgdir/models``.
 
+Taisei uses ``.obj`` for its 3D models. To export ``.obj`` files from Blender,
+use ``File -> Export -> Wavefront (.obj)``. Ensure that the following settings
+are used:
+
+::
+
+    Include
+        Objects as OBJ Objects: ENABLED
+
+    Transform
+        Forward: Y Forward
+        Up: -Z Up
+
+    Geometry
+        Write Materials: DISABLED
```

Comment:
Triangulation is important, Taisei will bail on quad faces. I'm pretty sure it's disabled by default, at least on my system.